src/ontology/pato-edit.obo

```diff
@@ -9594,7 +9594,6 @@ name: prominent
 def: "A quality inhering in a bearer by virtue of the bearer's standing out or projecting beyond a surface or line." [Dictionary.com:Dictionary.com]
 subset: value_slim
 is_a: PATO:0001598 ! protruding
-is_a: PATO:0015007 ! prominence
```

Comment:
why is prominent not anymore a subclass of prominence?

Reply:
Sorry - I think I misunderstood the request. I'll add it back, but should I move prominence to be a subclass of 'physical quality'?